src/unicode.cpp

```diff
@@ -39,8 +44,58 @@ constexpr bool to_lower_ascii(char* input, size_t length) noexcept {
   }
   return non_ascii == 0;
 }
-
-ada_really_inline constexpr bool has_tabs_or_newline(
+#if ADA_NEON
+ada_really_inline bool has_tabs_or_newline(
```

Comment:
What do you think about having `unicode_simd.cpp` file name convention, and moving all `simd` declarations there?

Reply:
Same answer as above. I recommend not overengineering at this time... until we know more. It is possible that we might end up using SIMD just for this one function, and creating a lot of extra code could add unnecessary complexity.

However, if we do add more SIMD support, we will have plenty of time then to add more code...